### ISSUE TITLE
RTL-SDR OGN: libjpeg package not found during installation

### ISSUE THREAD

**Issue by Flint4** [jprochazka/adsb-receiver#410]
Hi, 

during installation of RTL-SDR OGN the installation routine tries to install the package "libjpeg" which is not available in the Debian (Raspbian) Stretch repo as far as I can see (E: Unable to locate package libjpeg). That's why the adsb-receiver setup gets halted. 

Is there any additional repo required to get libjpeg?

Best wishes, 
Alex

**Comment by jprochazka** [comment]
Yes it does. I do plan on getting back to that script. libjpeg was renamed I think libjpeg8 because for some reason the people maintaing distro's, especially Unbuntu feel the need to make things more a pain in the ass... Anyways I had not heard from anyone since I first began initially created the Open Glider Network script which never made it as an option into a release back quite some time. I think @Romeo-Golf had the script modified to work at one point but I removed it from the options due to the exact issue you are having.

I am sorry to have not addressed this sooner and will work on getting a working script up and running as well as adding as an actual installation option once again within the next minor v2.x release. Keep in mind it will be a proof of concept process being it is pretty much a strictly Europian thing.

BTW how did you stumble accross the option to install it? LOL!
